data/json/recipes/basecamps/recipe_primitive_field.json

```diff
@@ -893,7 +893,6 @@
     "autolearn": false,
     "never_learn": true,
     "time": "510 m",
-    "blueprint_requires": [ { "id": "not_an_upgrade" } ],
```

Comment:
This seems intended: https://github.com/CleverRaven/Cataclysm-DDA/pull/34661#discussion_r334202993

Reply:
please don't take these out again.  It is needed to keep the blacksmith survey suppressed from the bulletin board menu per @mlangsdorf

Reply:
OK, will look into it more.

Reply:
This isn't removing it from the blacksmith entry, only from the faction wall entries, which I believe to be correct.

Reply:
I agree that it's not strictly needed for the faction wall entries, but I'd like to keep it anyway because it makes it clearer that they shouldn't go in the general upgrade missions.

Reply:
Well, I can't leave things how they are, because the tests will fail under this new regime.  I suggest either:
* Make it a 'commented out' field by renaming it `"//blueprint_requires"` or
* Tweak the loading code in `recipe.cpp` so that it visits the `blueprint_requires` field even when it ignores it.
I'm leaning towards the first option.  Any preference?

Reply:
So, what's the status quo here?

Reply:
I went for the 'commented out' option.  @mlangsdorf welcome to tweak as desired now that this is merged.